src/server/index_patterns/service/lib/index.js

```diff
@@ -1,3 +1,3 @@
 export { getFieldCapabilities } from './field_capabilities';
 export { resolveTimePattern } from './resolve_time_pattern';
-export { createNoMatchingIndicesError, isNoMatchingIndicesError } from './errors';
```

Comment:
The function was used more in the past, but I didn't remove it because it's used to ensure that `convertEsIndexNotFoundError()` is producing the right errors. Open to other solutions there and removing this if you have ideas. I would just much rather export `isNoMatchingIndicesError` than the `ERR_NO_MATCHING_INDICES` constant.